CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!--
+## [Unreleased]
+-->
+
+## [0.1.0] - 2024-??-??
+### Added
+- Initial release
+
+[Unreleased]: https://github.com/greatscottgadgets/packetry/compare/0.9.0...HEAD
+[0.1.0]: https://github.com/greatscottgadgets/packetry/releases/tag/0.9.0
```

Comment:
This currently refers to an `0.9.0` tag.

Reply:
Butterfingers! 🧈